users/users.md

```diff
@@ -1,8 +1,8 @@
 # User APIs
 
 [Get a user list](#get-a-user-list)  
-[Get a user](#get-a-user)  
-[Create user account](#create-user-account)  
+[Get a user](#get-a-user)<br />
```

Comment:
@WMKhant I think you should learn a bit about Markdown. It's not HTML. There are no tags like this.

Reply:
yep updated